airflow/providers/google/cloud/hooks/dataflow.py

```diff
@@ -1037,7 +1050,7 @@ def start_sql_job(
     def get_job(
         self,
         job_id: str,
-        project_id: str,
+        project_id: str = PROVIDE_PROJECT_ID,
```

Comment:
Is this related to other changes or just a drive-by improvement?

Reply:
Drive-by improvement